kolibri/plugins/coach/assets/src/views/common/assignments/SidePanelRecipientsSelector/LearnersSelectorSidePanel.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+
+  <SidePanelModal
+    alignment="right"
+    sidePanelWidth="700px"
+    closeButtonIconType="close"
+    @closePanel="$emit('close')"
+    @shouldFocusFirstEl="focusFirstEl"
+  >
+    <template #header>
+      <h1>
+        {{ $tr('selectGroupsAndIndividualLearnersTitle') }}
+      </h1>
+    </template>
+    <div style="padding-top: 30px">
```

Comment:
Once #12895 is merged we will need to remove this div, as its adding a fixing-padding that wont be needed after #12895